controllers/postgresqldatabase_controller.go

```diff
@@ -199,12 +219,32 @@ func stopRequeueOnInvalid(log logr.Logger, err error) error {
 	return nil
 }
 
-func (r *PostgreSQLDatabaseReconciler) EnsurePostgreSQLDatabase(log logr.Logger, host, name, user, password string, isShared bool) error {
-	credentials, ok := r.HostCredentials[host]
-	if !ok {
-		return &ctlerrors.Invalid{
-			Err: fmt.Errorf("unknown credentials for host"),
-		}
+// EnsureParams contains the required parameters for
+// `PostgreSQLDatabaseReconciler.EnsurePostgreSQLDatabase()`. It's exported to
+// match the method for which it's intended.
```

Comment:
I think we can skip the explanation of is availability. In practice I think the exporting of the method it self is some legacy stuff that we could clean up in another PR.

```suggestion
// `PostgreSQLDatabaseReconciler.EnsurePostgreSQLDatabase()`.
```

Reply:
Remove `exported...` comment